src/COVID19/model.py

```diff
@@ -176,6 +185,27 @@ def __init__(
             self.household_df = input_households
         elif not input_households:
             raise ParameterException("Household data must be supplied as a csv")
+        # if hospital_input_param_file:
+        #     self.c_params.hospital_input_param_file = hospital_input_param_file
+        # elif not hospital_input_param_file:
+        #     raise ParameterException("Hospital data must be supplied as a csv")
+        if hospital_param_line_number:
+            self.c_params.hospital_param_line_number = int(hospital_param_line_number)
+
+        if hospital_input_param_file and read_hospital_param_file:
+            self.c_params.hospital_input_param_file = hospital_input_param_file
+        elif not hospital_input_param_file and read_hospital_param_file:
+            raise ParameterException(
+                "Hospital param path is None and read hospital param file set to true"
+            )
+        else:
+            LOGGER.info(
+                "Have not passed hospital input file for params, use set_param or set_param_dict// crick todo look into this"
```

Comment:
Is this still TODO?

Reply:
Not sure what is meant by this - will check with the others.